src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/widget/WidgetPageObject.java

```diff
@@ -64,6 +68,9 @@ public WidgetPageObject createMultiple() {
     return this;
   }
 
+  /**
+   * Create incorrect widget tag on the tested article
```

Comment:
Are you sure these doc blocks are helping anyhow? The code is so simple and the variables/methods/object names are so clear, that for a developer it's obvious what's happening.